include/unicore-mx/usb/class/msc_scsi.h

```diff
@@ -0,0 +1,164 @@
+/** @defgroup usb_msc_scsi_defines USB MSC SCSI API  Definitions
+
+@brief <b>Defined Constants and Types for the USB MSC SCSI API Definitions</b>
+
+@ingroup USB_defines
+
+@version 1.0.0
+
+@author @htmlonly &copy; @endhtmlonly 2013
+alexrayne <alexraynepe196@gmail.com>
+@date 30 ecember 2016
+
+LGPL License Terms @ref lgpl_license
+*/
+/*
+ * Copyright (C) 2016 alexrayne <alexraynepe196@gmail.com>

```

Comment:
Minor note: INAL, as of my understanding of copyright.
copyright name/year/email for Weston Schmidt and/or Pavol Rusnak need to be added because they authored the file from which some of the content you moved.